packages/page-objects/src/components/sidebar/tree/debug/VariableSectionItem.ts

```diff
@@ -48,7 +49,7 @@ export class VariableSectionItem extends CustomTreeItem {
 	 * @deprecated For VS Code 1.88+ this method won't be working any more
 	 */
 	async getVariableNameTooltip(): Promise<string> {
-		if (VariableSectionItem.versionInfo.version >= '1.88.0') {
```

Comment:
any reason why AbstractElement needs replicate state of VSBrowser. requires managing duplicate state?

Reply:
when you init VSBrowser you copy the version over to AbstractElement but is there ever a situation where the version in AbstractElement would ever reflect differently. just seems like extra state to manage having to make sure AbstractElement reflects the actual version of vscode.

Reply:
I would need to double check, it can be some legacy from early stage of ExTester. The thing is `extester` and `page-objects` are separated packages. The `VSBrowser` instance storing a vscode version and driving execution is a parent which is loading during init phase appropriate set of page objects depending on VS Code version. Than the version is passed and stored also in `page-objects`. So I agree the version is same, but the point is to init page objects for a version of running browser.